drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -468,6 +477,13 @@ local function do_configure(driver, device)
       profile_name = profile_name .. level_name .. "-level"
     end
 
+    -- for devices with a feature map that indicate they support WindMode,
+    -- but who offer no support their WindSupport attribute.
+    local num_wind_modes = device:get_field(NUM_SUPPORTED_WIND_MODES)
```

Comment:
I think this code is run before the code that sets the field so it's always coming back as nil